### DISCUSSION TITLE
webrtc_server

### DISCUSSION THREAD

**Discussion by SShackSS** [isl-org/Open3D#6293] [Q&A]
Hi,
I build a code in Ubuntu 22.04 from open3D v0.16.1 example folder. The example named DrawWebRTC.cpp . It compiles with no errors. When I run it I see:

[Open3D INFO] WebRTC GUI backend enabled.
[Open3D INFO] Window window_0 created.
[Open3D INFO] EGL headless mode enabled.
[Open3D INFO] ICE servers: .....
FEngine (64 bits) created at 0x7f6c4ad84010 (threading is enabled)
[Open3D INFO] Set WEBRTC_STUN_SERVER environment variable add a customized WebRTC STUN server.
[Open3D INFO] WebRTC HTTP server handshake mode enabled.
[Open3D INFO] Open3D WebVisualizer is serving at http://localhost:8888.
[Open3D INFO] Set WEBRTC_IP and WEBRTC_PORT environment variable to customize the HTTP server address.
EGL(1.5)
OpenGL(4.2)
[Open3D DEBUG] LoadMaterialFromFile(): /home/shacks/development/resources/defaultLit.filamat
[Open3D DEBUG] LoadMaterialFromFile(): /home/shacks/development/resources/defaultLitTransparency.filamat
[Open3D DEBUG] LoadMaterialFromFile(): /home/shacks/development/resources/defaultLitSSR.filamat
[Open3D DEBUG] LoadMaterialFromFile(): /home/shacks/development/resources/defaultUnlit.filamat
[Open3D DEBUG] LoadMaterialFromFile(): /home/shacks/development/resources/defaultUnlitTransparency.filamat
[Open3D DEBUG] LoadMaterialFromFile(): /home/shacks/development/resources/depth.filamat
[Open3D DEBUG] LoadMaterialFromFile(): /home/shacks/development/resources/unlitGradient.filamat
[Open3D DEBUG] LoadMaterialFromFile(): /home/shacks/development/resources/normals.filamat
[Open3D DEBUG] LoadMaterialFromFile(): /home/shacks/development/resources/unlitSolidColor.filamat
[Open3D DEBUG] LoadMaterialFromFile(): /home/shacks/development/resources/unlitBackground.filamat
[Open3D DEBUG] LoadMaterialFromFile(): /home/shacks/development/resources/infiniteGroundPlane.filamat
[Open3D DEBUG] LoadMaterialFromFile(): /home/shacks/development/resources/unlitLine.filamat
[Open3D DEBUG] LoadMaterialFromFile(): /home/shacks/development/resources/unlitPolygonOffset.filamat
[Open3D DEBUG] Trying to destroy default resource [Texture, 1, hash: 655361]. Nothing will happen.
[Open3D DEBUG] Trying to destroy default resource [Texture, 1, hash: 655361]. Nothing will happen.
[Open3D DEBUG] Trying to destroy default resource [Texture, 1, hash: 655361]. Nothing will happen.
[Open3D DEBUG] Trying to destroy default resource [Texture, 1, hash: 655361]. Nothing will happen.
[Open3D DEBUG] Trying to destroy default resource [Texture, 1, hash: 655361]. Nothing will happen.

When I open http://localhost:8888/ There is a text there “Error 404: Not found”. What did I miss? Why it doen’t work properly?

**Comment by ssheorey** [comment]
Hi @SShackSS you also need to copy over html resources to the install location.
For example see: 
https://github.com/isl-org/Open3D/tree/master/cpp/open3d/visualization/webrtc_server/html
